### DISCUSSION TITLE
How to manage versions of the same template/os.

### DISCUSSION THREAD

**Discussion by rahmnstein** [vmware/packer-examples-for-vsphere#983] [Help] [answered]
When I build a template, it ends up being named something like linux-oracle-9.1-HEAD, with HEAD coming from Git.

How are you dealing with versioning of templates? What happens when I want a new version of the same template just with the latest patches? It will still be called linux-oracle-9.1-HEAD and I would have to replace the old one.

Do you manually rename all templates afterwards? I would like to have like a linux-oracle-9.1-<year><month> but that would require me to change main pkr file linux-oracle.pkr.hcl which makes it harder to fetch changes from this repo.

How are you doing it ? Rename the template afterwards manually or in your CI or just keep the latest of every OS? Something I miss that can achieve what I want?

**Comment by tenthirtyam** [comment, marked as answer]
You can checkout a new branch (e.g., 2024-11-21) and it wouod append that to the template during the run.

Other improvements are planned but Inoony get to them in my personal time lately.
